pkg/resource/aws/aws_security_group_rule_ext.go

```diff
@@ -128,5 +128,5 @@ func (r *AwsSecurityGroupRule) String() string {
 	if r.PrefixListIds != nil && len(*r.PrefixListIds) > 0 {
 		attrs = append(attrs, fmt.Sprintf("%s: %s", sourceOrDestination, strings.Join(*r.PrefixListIds, ", ")))
 	}
-	return strings.Join(attrs, ", ")
+	return fmt.Sprintf("%s\n        %s", r.CreateIdHash(), strings.Join(attrs, ", "))
```

Comment:
this `CreateIdHash` was definitively missing, thanks!

Reply:
I'm reworking this PR, ID will be print every time for each resource when i'm done.